demisto_sdk/commands/validate/file_validator.py

```diff
@@ -509,6 +511,12 @@ def validate_added_files(self, added_files, file_type: str = None, modified_file
             if re.search(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) and not file_type:
                 continue
 
+            elif ('ReleaseNotes' in file_path) and not self.skip_pack_rn_validation:
```

Comment:
shouldnt it be releasenotes ?

Reply:
@amshamah419 - I believe this is the unmodified file path so it should be RealseNotes - right?

Reply:
Correct. It should be `ReleaseNotes`.